doc/usermanual/darkroom/modules/tone/filmicrgb.xml

```diff
@@ -0,0 +1,619 @@
+        </para></listitem>
+
+        <listitem><para>
+          The
+          <guilabel>display</guilabel>
+          tab defined the output settings to map the transformed image to the display. In
+          typical use cases, this tab should only be used very rarely.
+        </para></listitem>
+
+      </itemizedlist>
+    </para>
+
+    <para>
+      The sliders' ranges of filmic rgb are limited to usual and safe values, but values are
+      allowed out of these ranges by clicking on the sliders with the right button and inputting
+      values on the keyboard. Filmic rgb has no neutral parameters resulting in a no-operation:
+      as soon as the module is enabled, the image is always at least slightly affected.
+    </para>
+
+    <para>
+      The curves at the top of the module are read-only and serve as a guide for the operations
+      performed on the sliders. The bright curve is the tone mapping curve, where the abscissa
+      represents the scene exposure, and the ordinate represents the display exposure. The dark
+      curve is the desaturation curve, representing the percentage of saturation as function of
+      the scene exposure.
+    </para>
+
+    <sect5>
+      <title>Middle-grey luminance (<guilabel>Scene</guilabel> tab)</title>
+<!-- IHU - okay, I'm not clear on color spaces. My understanding is that filmic
+        worked on XYZ and filmic rgb works on RGB - is that true? does that sentence make sense?
+        :) -->
+      <para>
+        The middle-grey luminance is the luminance in RGB space of the scene referred 18% grey.
+        Its color picker tool reads the average luminance over the drawn area. If you happen to
+        have a grey card or a color chart (IT8 chart or colorchecker) shot in the scene lighting
+        conditions, then the grey color picker tool can be used to quickly sample the luminance
+        of the grey patch on the picture. In other situations, the color picker can be used to
+        sample the average luminance of the subject.
+      </para>
+      <para>
+        This setting has an effect on the picture that is analogous to a lightness correction.
+        Values close to 100% do not compress the highlights but fail to recover shadows. Values
+        close to 0% recover greatly the shadows but compress the highlights more harshly and
+        result in local-contrast losses. The standard middle-grey value for linearly encoded
+        camera RGB is 18%. Good values of grey are usually the average luminance of the whole
+        picture or of the subject. In studio and indoors (low dynamic range scenes), proper grey
+        values are found between 15-18%. In high dynamic range scenes (landscapes, back-lit
+        portraits), proper grey values lie between 1.25 and 9%.
+      </para>
+      <para>
+        When modifying the middle-grey luminance, the white and black exposures are
+        automatically slided accordingly, to preserve the dynamic range from clipping and to
```

Comment:
The correct past-tense of 'slide' is 'slid', not 'slided'. :)

automatically slid accordingly, to preserve the dynamic range from clipping and to